src/actions/write/submitAuctionBid.ts

```diff
@@ -89,23 +85,20 @@ export const submitAuctionBid = (
   assertAvailableRecord({
```

Comment:
This function could benefit from unit tests.

Reply:
The same goes for the functions it uses internally:
`isExistingActiveRecord`
`isActiveReservedName`
`isShortNameRestricted`

Reply:
i'll add in tick-state branch. the intent of this PR was to get unit tests properly setup for future additions.